### PR TITLE
Add new History Actions and add Process Types - plus stylecop issues

### PR DESCRIPTION
**Pull request by deevezina**

### CODE REVIEW

**Comment by deevezina** [comment]
Add additional history actions and add column for process types

File: src/Translations/TranslationIntegrationModel.cs [hunk 1]
```diff
@@ -133,7 +133,7 @@ public class TranslationIntegrationModel
         public MicroUserModel LastUpdatedByUser { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether this <see cref="TranslationIntegration"/> is active.
+        /// Gets or sets a value indicating whether this <see cref="TranslationIntegrationModel"/> is active.
```
Comment (ahussain-vasont) [inline src/Translations/TranslationIntegrationModel.cs:136]: Not sure if that <see cref> is needed here, it's referencing to the same model anyways.

File: src/Versioning/ChangesetSnapshotModel.cs [hunk 2]
```diff
@@ -52,7 +52,31 @@ public enum ChangesetSnapshotActionType
...
+        Branch,
+
+        /// <summary>
+        /// Create a translation
+        /// </summary>
+        [Description("t")]
+        Translate,
+
+        /// <summary>
+        /// Create a publish
```
Comment (ahussain-vasont) [inline src/Versioning/ChangesetSnapshotModel.cs:70]: "Create a publish" doesn't seem correct. May be a "Publish action type"?

**Review by ahussain-vasont — APPROVED** [review]
The 2 comments I left are very minor, so approving it, if you wanted to make those changes you can push further updates, if not that's fine as well.

File: src/Translations/TranslationIntegrationModel.cs (hunk 1 above)
Comment (deevezina) [reply src/Translations/TranslationIntegrationModel.cs:136]: Do you care if I remove the entire "<see/>" so it doesn't cause a stylecop error?

**Review by ahussain-vasont — APPROVED** [review]
Much better, and could help others understand it a bit better.

**Pull request merged**